package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "to-do-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "tsc index.ts && node dist/index.js && rm index.js"
+  },
```

Comment:
We could use something called `nodemon` here to automatically watch for changes in our TS files and then run the compiler when it changes